resources/js/api/base.js

```diff
@@ -48,6 +48,8 @@ export default {
       vm.$router.replace({ name: 'home' });
     } else if (responseStatus === env.HTTP_PAYLOAD_TOO_LARGE) { // 413 => payload to large
       vm.flashMessage.error(vm.$t('app.flash.tooLarge'));
+    } else if (responseStatus === env.HTTP_SERVICE_UNAVAILABLE) { // 503 => maintenance mode
+      window.location.reload();
```

Comment:
Would it be better to delay the reload? Otherwise to many requests would be send.

Reply:
It would be better to have a maintenance page for this case!

Reply:
There is no message just a page reload, this is not intuitive!

Reply:
On other errors that not handled here, the page will be broken, since the room card will have an overlay with a loading indicator!